pages.es/common/nms.md

```diff
@@ -0,0 +1,20 @@
+# nms
+
+> Herramienta de línea de comandos que recrea el famoso efecto de desencriptado de datos de la película Sneakers (1992).
+> Más información: <https://github.com/bartobri/no-more-secrets>.
+
+- Desencriptar texto tras presionar una tecla:
+
+`echo "{{Hola, Mundo!}}" | nms`
+
+- Desencriptar output, inmediatamente, sin esperar a que una tecla sea pulsada:
```

Comment:
Aunque a mí y a ti nos suene mal `la salida`, hay libros de informatica en Español así :man_shrugging:  : 
```suggestion
- Desencriptar la salida inmediatamente, sin esperar a que una tecla sea pulsada:
```
`la salida` is more closer to `the exit` in English :joy:, but the Spanish books that I have read use that translation :man_facepalming:

Reply:
Well, if textbooks say so, who am I to disagree? 😅

Reply:
That's why I don't read tech stuff in Spanish.